packages/webview_flutter/webview_flutter_android/CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to `webview_flutter_android`
+
+Please start by taking a look at the general guide to contributing to the `flutter/packages` repo:
+https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
+
+## Package Structure
+
+This plugin serves as a platform implementation plugin as outlined in [federated plugin](https://docs.flutter.dev/packages-and-plugins/developing-packages#federated-plugins). 
+The sections below will provide an overview of how this plugin implements this portion with Android.
+
+For making changes to this package, please take a look at [changing federated plugin](https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changing-federated-plugins).
+
+### Quick Overview
+
+This plugin implements the platform interface provided in `webview_flutter_platform_interface` using
+the native WebKit APIs for Android.
+
+#### SDK Wrappers
+
+To access native APIS, this plugins uses Dart wrappers of the native library. The native library is
+wrapped using using the `ProxyApi` feature from the `pigeon` package.
+
+The wrappers for the native library can be updated and modified by changing `pigeons/android_webview.dart`.
+
+The generated files are located:
+* `lib/src/android_webkit.g.dart`
+* `android/src/main/java/io/flutter/plugins/webviewflutter/AndroidWebkitLibrary.g.kt`
+
+To update a wrapper for a platform, follow the steps:
```

Comment:
"a wrapper for a platform" -> "the wrapper" (since this package is single-platform).